src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudTest.java

```diff
@@ -61,8 +61,9 @@ public void construction() throws Exception {
     serviceAccountConfig.setSecretJsonKey(bytes);
     assertNotNull(serviceAccountConfig.getAccountId());
     // Create a credential
-    Credentials c =
-        (Credentials) new GoogleRobotPrivateKeyCredentials(ACCOUNT_ID, serviceAccountConfig, null);
+    GoogleRobotPrivateKeyCredentials c =
+        new GoogleRobotPrivateKeyCredentials(ACCOUNT_ID, serviceAccountConfig, null);
+    String credentialsId = c.getId();
```

Comment:
is this not deprecated - should you not be using the non deprecated constructor and passing in an ID here?

Reply:
https://github.com/jenkinsci/google-compute-engine-plugin/pull/416#issuecomment-1779302755